packages/dma-contracts/scripts/deployment/deploy.ts

```diff
@@ -177,14 +178,42 @@ export class DeploymentSystem extends DeployedSystemHelpers {
     }
   }
 
+  findPath = (obj, target, parentPath) => {
+    for (const key in obj) {
+      const path = `${parentPath}.${key}`
+      if (typeof obj[key] === 'string' && obj[key] === target) {
+        return path
+      }
+      if (typeof obj[key] === 'object') {
+        const result = this.findPath(obj[key], target, path)
+        if (result) {
+          return result
+        }
+      }
+    }
+    return null
+  }
+
+  findStringPath = target => {
+    const rootPath = 'CONTRACT_NAMES'
+    return this.findPath(CONTRACT_NAMES, target, rootPath)
+  }
+
+  replaceServiceRegistryName(inputString, transformFunction) {
+    return inputString.replace(/(serviceRegistryName:\s')([^']*)(')/g, function (match, p1, p2) {
+      const newValue = transformFunction(p2)
+      return 'serviceRegistryName: ' + newValue
+    })
+  }
+
   async saveConfig() {
     const { writeFile } = await import('fs')
-
-    const configString = inspect(this.config, { depth: null })
+    let configString = inspect(this.config, { depth: null })
+    configString = this.replaceServiceRegistryName(configString, this.findStringPath)
 
     writeFile(
-      this.getConfigPath(`./${this.network}.conf.ts`),
-      `export const config = ${configString}`,
+      `./../deploy-configurations/configs/${this.network}.conf.ts`,
+      `import { SystemConfig } from '@deploy-configurations/types/deployment-config'\nimport { CONTRACT_NAMES } from '@deploy-configurations/constants'\n\nexport const config: SystemConfig = ${configString} \n`,
```

Comment:
Maybe we could extract the initial string into it's own variable. So, it's obvious that it's top of file imports?